src/Knp/Rad/ViewRenderer/Renderer/ControllerRenderer.php

```diff
@@ -62,7 +62,7 @@ public function renderResponse($data = null)
 
         $template = $this->buildTemplate($request);
 
-        if (false === $this->isMasterRequest($request)) {
+        if (false === $this->isMasterRequest($request) || $request->isXmlHttpRequest()) {
```

Comment:
if (false === $this->isMasterRequest($request) || **true ===** $request->isXmlHttpRequest())